hoot-core/src/main/cpp/hoot/core/conflate/poi-polygon/PoiPolygonMatch.cpp

```diff
@@ -486,6 +486,11 @@ void PoiPolygonMatch::calculateMatch(const ElementId& eid1, const ElementId& eid
   LOG_TRACE("**************************");
 }
 
+MatchType PoiPolygonMatch::getType() const
+{
+  return _threshold->getType(_class);
+}
+
```

Comment:
This overridden version of `getType` gets the match/miss/review type from the `MatchClassification` object in this class and not in the `Match` base class.